cf-monitord/history.c

```diff
@@ -470,7 +473,7 @@ static PromiseResult NovaExtractValueFromStream(EvalContext *ctx, const char *ha
 
         if (a->measure.select_line_matching && StringMatchFull(a->measure.select_line_matching, ip->name))
         {
-            Log(LOG_LEVEL_VERBOSE, " ?? Look for %s regex %s", handle, a->measure.select_line_matching);
+            Log(LOG_LEVEL_VERBOSE, "  Found regex '%s' matches line '%s'", a->measure.select_line_matching, ip->name);
```

Comment:
Why does the log message start with spaces?

Reply:
I don't know for sure, my guess is to make it indent for readability in the multiline logs. I just retained the existing 2 spaces.

I suppose if we wanted to support log indention we could add a parameter to the Log function.